### PR TITLE
Set text compress as default

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

Set text compress as default due to https://github.com/fluent/fluentd/issues/3110

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -211,6 +211,7 @@ fluentd:
     totalLimitSize: "128m"
     retryMaxInterval: "10m"
     retryForever: true
+    compress: text
```
Comment (frankreno) [inline deploy/helm/sumologic/values.yaml:214]: Should we add a comment that GZIP is not supported so it is clear why we set it this way?

Comment (sumo-drosiek) [reply deploy/helm/sumologic/values.yaml:214]: @frankreno I would say not recommended and point to the issue. Is it ok?

**Comment by sumo-drosiek** [comment]
@vsinghal13 it was 10 times for my test data. It will be compression rate of the gzip for the given log files

**Review by perk-sumo — COMMENTED** [review]
LGTM, thanks!

**Pull request merged**